QuickSwiftCheck.xcodeproj/project.pbxproj

```diff
@@ -280,13 +280,16 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 3AD546101F17A0D50000CCCA /* Common-Debug.xcconfig */;
 			buildSettings = {
+				ENABLE_TESTING_SEARCH_PATHS = YES;
 			};
 			name = Debug;
 		};
 		3A5222451F13E02A005D985A /* Release */ = {
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 3AD546111F17A0D50000CCCA /* Common-Release.xcconfig */;
 			buildSettings = {
+				ENABLE_TESTING_SEARCH_PATHS = YES;
+				SWIFT_COMPILATION_MODE = wholemodule;
```

Comment:
WMO wasn't enabled for a reason. QuickSwiftCheck is a testing framework. Leaving it unoptimized will maximize debugging capabilities even if tests run in release configuration. Please ignore Xcode's suggestion and remove this setting.